besu/src/main/java/org/hyperledger/besu/cli/subcommands/networkcreate/mapping/MapperAdapter.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.hyperledger.besu.cli.subcommands.networkcreate.mapping;
+
+import org.hyperledger.besu.cli.subcommands.networkcreate.model.Configuration;
+import org.hyperledger.besu.ethereum.core.Address;
+import org.hyperledger.besu.ethereum.core.Wei;
+
+import java.security.InvalidParameterException;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser.Feature;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.google.common.io.Files;
+
+// TODO Handle errors
+public abstract class MapperAdapter {
+
+  ObjectMapper mapper;
```

Comment:
(question) why this field ?

Reply:
because I need to be able to access the mapper later, without creating a new one when I call writeValueAsString